server/apps/users/filters.py

```diff
@@ -0,0 +1,30 @@
+import django_filters
+
+from apps.contact.models import Contact
+from django.contrib.auth.models import Group
+
+
+class UserFilter(django_filters.FilterSet):
+    role = django_filters.CharFilter(method='filter_role')
+    roleIn = django_filters.CharFilter(method='filter_role_in')
+
+    class Meta:
+        model = Contact
+        fields = ['email']
+
+    def filter_role(self, queryset, name, value):
+        if not value:
+            return queryset
+        group = Group.objects.get(name__iexact=value)
+        return queryset.filter(
+            groups=group
+        ).distinct()
```

Comment:
This will throw error if name with value is not found. We should use something like this.
```suggestion
        return queryset.filter(
            groups__name__iexact=value
        ).distinct()
```